src/WebJobs.Script.Host/Program.cs

```diff
@@ -1,34 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using Microsoft.Azure.WebJobs.Script.Config;
-
-namespace Microsoft.Azure.WebJobs.Script.Host
-{
-    public static class Program
-    {
-        public static void Main(string[] args)
```

Comment:
Was this meant to be a sample or an alternative host for debugging as a console app? If we want a sample, it would probably be nice to have something better showing different patterns.

Reply:
The goal is to have a sample host that can be pointed at a functions code directory and run. E.g. execute queue trigger functions, etc. Can you just migrate this forward?